ecodev_core/custom_equal.py

```diff
@@ -25,3 +25,10 @@ def custom_equal(element_1: Optional[object], element_2: Optional[object], eleme
         return False
 
     return np.isclose(element_1, element_2) if element_type == float else element_1 == element_2
+
+
+def is_null(value: Any) -> bool:
```

Comment:
you can put it in pandas utils, given that's it is not really a custom equal, and used there :)